app/services/create_batch.rb

```diff
@@ -0,0 +1,12 @@
+class CreateBatch
+  def call
+    appointment_summaries = AppointmentSummary
+                            .includes(:appointment_summaries_batches)
+                            .where('appointment_summaries_batches.appointment_summary_id': nil)
+                            .references(:appointment_summaries_batches)
```

Comment:
I think a comment of what this actually does is a good idea. I could understand it more when it was SQL!